### DISCUSSION TITLE
Feature Request: Conditional View

### DISCUSSION THREAD

**Discussion by rillekille** [home-assistant/frontend#9509] [Dashboard]
I'm not sure if this is possible or not but it would be very nice if a view could be based on conditions.
For example visible when a person/device tracker has status home and hidden if the status is not_home.

I tested with a conditional card and it worked very well but it would be much simpler if it could be handled on the view level.
If the status is home then all cards in the view should be visible (unless you hace a conditional card in the view which shouldn't be visible). And vice versa, if the status is not_home the view should be hidden.

Regards

/Rickard

**Comment by Mariusthvdb** [comment]
big +1 to this, and it being related to the https://community.home-assistant.io/t/feature-request-conditional-views/320508 community FR.

Since the custom option we had for doing this (using custom-header) now is no longer functional, a core solution would be all the more appreciated.. it's a big gap in the toolbox really.

besides the user-id/admin condition, please allow any condition, by 'simply' adding the conditional option like we have for the cards to views....

hope this can be considered

**Comment by finity69x2** [comment]
Yes, it would definitely be very much appreciated if this could be implemented. 

It seems the framework is already there as noted above for the conditions so it should be just adding entity states as conditions as well.
